urdf_mesh_converter/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(urdf_mesh_converter)
+
+Find_package(
+  catkin REQUIRED
+  catkin_virtualenv
+)
+
+if (${catkin_virtualenv_VERSION} VERSION_LESS "0.5.1")
+  message(STATUS "${PROJECT_NAME} requires catkin_virtualenv >= 0.5.1")
```

Comment:
I think `message(WARNING ...` is easier to notice.

Reply:
Thanks. Fixed.
https://github.com/jsk-ros-pkg/jsk_model_tools/pull/255/commits/b64b866662abb42d2622789f247704933559c595